data/json/recipes/recipe_electronics.json

```diff
@@ -673,6 +674,7 @@
       [ "scrap", 6 ],
       [ "pan", 2 ],
       [ "pot", 2 ],
+      [ "pot_copper", 2 ],
       [ "knife_butcher", 6 ],
       [ "knife_steak", 18 ],
       [ "knife_butter", 18 ],
```

Comment:
I just noticed how idiotic this recipe is. 18 steak knives? Two whole pots (each larger than the complete maker)?